packages/lodestar/test/unit/chain/attestation/pool.test.ts

```diff
@@ -35,17 +38,19 @@ describe("attestation pool", function () {
 
   it("should not receive old attestation", async function () {
     const attestation = generateAttestation({data: {target: {epoch: 0}}});
-    chain.getGenesisTime.returns(
+    (chain.clock as any)._currentSlot = getCurrentSlot(
```

Comment:
:vomiting_face:  :rofl: 

stub getter function.
I think it's something like `sinon.stub(chain.clock, "currentSlot").get(() => 15)`

Reply:
:+1:  will try that

Reply:
for some reason this doesn't work. It sets the stub getter appropriately, but isn't set when called thru another module :thinking:

Reply:
the unit tests also seem to be hanging bc in `test/unit/sync/initial/fast.test.ts`, the stubbed `networkStub.getPeers` isn't being used, and its causing an unhandled promise rejection.